examples/asynchronous_processing.rs

```diff
@@ -66,14 +65,14 @@ fn run_async_processor(brokers: &str, group_id: &str, input_topic: &str, output_
         .create()
         .expect("Producer creation error");
 
-    // Create the thread pool where the expensive computation will be performed.
-    let thread_pool = thread_pool::Builder::new()
+    // Create the runtime where the expensive computation will be performed.
+    let mut thread_pool = tokio::runtime::Builder::new()
         .name_prefix("pool-")
-        .pool_size(4)
-        .build();
+        .build()
```

Comment:
For the sake of parity with the previous example, would it make sense to keep the `.pool_size(4)` here?

Reply:
Oh, yeah, looks like the option still exists, but it's been renamed to `core_threads`, which is why I didn't spot it. Updated!